dockstore-webservice/src/main/java/io/dockstore/webservice/core/database/UserInfo.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2021 OICR and UCSC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.dockstore.webservice.core.database;
+
+public class UserInfo {
+
+    private String dockstoreUsername;
+    private String thirdPartyUsername;
```

Comment:
I would have preferred something like:

```
private String dockstoreUsername;
private String gitHubUsername;
private String gitHubEmail;
private String googleUsername;
private String googleEmail; // Actually, not sure if google username exists and/or of it can be different than the email.
```

But your way works, and is more flexible if we add more logins (you wouldn't need to change the structure).